### DISCUSSION TITLE
[@types/connect-redis] Why db field shouldn't be string instead of number in RedisStoreOptions?

### DISCUSSION THREAD

**Discussion by AndreiF13** [DefinitelyTyped/DefinitelyTyped#56156] [Issues with a @types package] [answered]
Hi!
In @types/connect-redis, under the interface RedisStoreOptions there is a field "db" which has the type number instead of string. In my opinion should be string because it is for defining the database name.

Thank you.
Best regards!

[a comment by an automated account is not shown]

**Comment by JipSterk** [comment, marked as answer]
I've looked into what the type is underlying packages like `ioredis` here its of the type [number](https://github.com/luin/ioredis/blob/f02e383996310adaefc2b4c40d946b76e450e5c7/lib/redis/RedisOptions.ts#L20)
i have not used `node-redis` myself but couldn't find any field named `db`
i did come across [`selectedDB`](https://github.com/NodeRedis/node-redis/blob/c19d200b91c3c32fde7bf75c859daf9b416a5540/lib/client.ts#L123) but its also of the type number.

i did find [this](https://redis.io/commands/select) on the redis docs

> Select the Redis logical database having the specified zero-based numeric index. 
New connections always use the database 0.

**Reply by AndreiF13** [reply]
Thank you!
